demo/index.js

```diff
@@ -47,6 +47,16 @@ var plugin = function (config) {
 
     isRemovalOfLastAssocListItemAllowed(/*field*/) {
       return false;
+    },
+
+    createElement_Fieldset: function (props, ...children) {
```

Comment:
@jdeal can you take a look at my updated example? I had to do a bit more work than I had expected, tell me if it makes sense :/